examples/Building-and-deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb

```diff
@@ -90,36 +90,28 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    from merlin.core.dispatch import get_lib
</pre><p>Is this imported function used anywhere? If not, it could be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
let me check. thanks.